server.py

```diff
@@ -0,0 +1,23 @@
+import socket
+
+
+def main():
+    serv = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    serv.bind(('localhost', 50000))
+    serv.listen(5)
+    conn, addr = serv.accept()
+    from_client = ''
+    data = "Start"
+    while data != "":
+        data = conn.recv(4096)
+        if not data:
+            break
+        from_client += data
```

Comment:
Why are you replying with everything the client has sent so far?
If I sent `1` then `2`
I'm going to get `1212`, instead of just `22`